src/config/misc.toml

```diff
@@ -7,3 +7,6 @@ fetch = "origin"
 
 [revset-aliases]
 # Placeholder: added by user
+
+[ui]
+log-word-wrap = false
```

Comment:
Why did you decide to make this `false` by default?

After some testing period, I'd lean towards making it true by default eventually. I'm curious if you agree with that.

Reply:
It's just my preference. I use `less -RS`.

I'm not against changing the default if that's preferred.